nrich-excel/src/test/java/net/croz/nrich/excel/testutil/ExcelReportRequestGeneratingUtil.java

```diff
@@ -12,15 +11,15 @@ private ExcelReportRequestGeneratingUtil() {
     }
 
     public static CreateExcelReportRequest createExcelReportRequest(Object[][] rowData, int batchSize, OutputStream outputStream, int firstRowIndex) {
-        MultiRowDataProvider multiRowDataProvider = (start, limit) -> start == 0 ? rowData : null;
-
-        return createExcelReportRequest(rowData == null ? null : multiRowDataProvider, batchSize, outputStream, firstRowIndex);
+        return CreateExcelReportRequest.fromFlatData(rowData)
+            .batchSize(batchSize)
+            .outputStream(outputStream)
+            .templatePath("classpath:excel/template.xlsx")
```

Comment:
Optional: we could move this path String to a TEMPLATE_PATH constant, as we have it 2 times now in this class.